TimeTracker/LSITimedTask.h

```diff
@@ -0,0 +1,28 @@
+//
+//  LSITimedTask.h
+//  TimeTracker
+//
+//  Created by Kelson Hartle on 7/14/20.
+//  Copyright © 2020 Lambda School. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface LSITimedTask : NSObject
+
+@property NSString *clientName;
+@property NSString *workSummary;
+@property int hourlyRate;
+@property int amountOfHoursWorked;
```

Comment:
You would use copy when you want to guarantee the state of the object.
```suggestion
@property (nonatomic, copy) NSString *clientName;
@property (nonatomic, copy) NSString *workSummary;
@property (nonatomic) double hourlyRate;
@property (nonatomic) double hoursWorked;
```